src/named_systems2.jl

```diff
@@ -401,15 +428,15 @@ The following complicated feedback interconnection
 
 ```
                  yF
-              ┌────────────────────────────────┐
-              │                                │
-    ┌───────┐ │  ┌───────┐ yR   ┌─────────┐    │    ┌───────┐
-uF  │       │ │  │       ├──────►         │ yC │  uP│       │    yP
-────►   F   ├─┴──►   R   │      │    C    ├────+────►   P   ├────┬────►
-    │       │    │       │   ┌──►         │         │       │    │
-    └───────┘    └───────┘   │  └─────────┘         └───────┘    │
-                             │                                   │
-                             └───────────────────────────────────┘
+               ┌────────────────────────────────┐
+               │                                │
+    ┌───────┐  │  ┌───────┐        ┌───────┐    │    ┌───────┐
+uF  │       │  │  │       | yR     │       │ yC │ uP │       │   yP
+────►   F   ├──┴──►   R   │────┬───►   C   ├────┴────►   P   ├───┬────►
```

Comment:
```suggestion
────►   F   ├──┴──►   R   │────+───►   C   ├────+────►   P   ├───┬────►
```
it probably makes sense to keep the `+`es?